includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -816,6 +816,7 @@ public function ajax_get_selected_product_data() {
 
 			if ( 'variable' === ( WC_Stripe_Helper::is_wc_lt( '3.0' ) ? $product->product_type : $product->get_type() ) && isset( $_POST['attributes'] ) ) {
 				$attributes = array_map( 'wc_clean', $_POST['attributes'] );
+				$attributes = array_map( 'stripslashes_deep', $attributes );
```

Comment:
@hardipparmar-multidots, I'm concerned with this change because it removes slashes after attribute value has been sanitized with `wc_clean`. It's not that secure IMO, so I'd request second opinion here. @RadoslavGeorgiev what do you think on this?

Reply:
This pattern ( `wc_clean( wp_unslash( $_POST['whatever'] ) )` ) [is being used already](https://github.com/woocommerce/woocommerce/blob/07ce4b9e1cbb0ab736ce04e51e602ae8557f3ba0/includes/admin/class-wc-admin-attributes.php#L68) (`wp_unslash` is the same as `stripslashes_deep`). These attributes aren't being used as part of a MySQL query, so there's no security risk.

Reply:
@DanReyLop, thanks for checking in! As I can see, the order is different:

1. Proposed change: clean => remove_slashes => result.
2. The pattern you've mentioned: remove_slashes => clean => result. 

In the first case, the result will miss slashes added by `clean` and that was my concern.

> These attributes aren't being used as part of a MySQL query, so there's no security risk.

That's a good argument and removes my concern 🙂

Reply:
> That's a good argument and removes my concern 🙂

haha yeah, it removed my concern too. To be honest, I'm not sure we even need the `wc_clean`, but it makes sense to leave it there for historical purposes.